src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -891,6 +891,80 @@ def get_frames_per_slice(file_path: PathType) -> int:
         frames_per_slice = non_varying_frame_metadata.get("SI.hStackManager.framesPerSlice", 1)
         return frames_per_slice
 
+    def get_original_frame_indices(self, plane_index: Optional[int] = None) -> np.ndarray:
+        """
+        Get the original frame indices for each sample.
+
+        Returns the index of the original frame for each sample, mapping processed samples
+        back to their corresponding frames in the raw microscopy data. This accounts for
+        any filtering, subsampling, or exclusions (such as flyback frames) performed by
+        the extractor.
+
+        Parameters
+        ----------
+        plane_index : int, optional
+            Which plane to use for frame index calculation in volumetric data.
+            If None, uses the convention of using the last plane as its the default for
```

Comment:
This seems like a bit of a strange default. Is there a reason for this choice?

Reply:
This is the one that we used for default in volumes.

Reply:
You're right. This isn't clear enough. Let me improve this.

As we've discussed before, the timestamp assigned to each volumetric sample corresponds to the last frame of the volume. That's the behavior currently implemented:

https://github.com/catalystneuro/roiextractors/blob/2a94b760ab4944851136d74d77e0fa9f8d999ba9/src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py#L753-L779

So, when I am extracting the frame that corresponds to a volume sample I am just using the same policy in reverse but also providing flexibility. 

Does that makes sense?

Reply:
I think this reveals that the docstrings should be improved though.